test/playwright.config.ts

```diff
@@ -0,0 +1,15 @@
+// eslint-disable-next-line @typescript-eslint/consistent-type-imports
+import type { PlaywrightTestConfig } from '@playwright/test';
+
+const config: PlaywrightTestConfig = {
+  reporter: 'html',
+  webServer: {
+    command: 'node test/serve.js',
+    url: 'http://localhost:3000',
+    reuseExistingServer: !process.env.CI,
+  },
+  use: {
+    baseURL: 'http://localhost:3000',
```

Comment:
I’m new to Playwright. Do we have to specify `webServer.url` and `use.baseURL`? If so, maybe this can be specified in a reused variable.

Reply:
I’m new to Playwright too (and tests at all)! This PR is my "getting started" :) I was following [the docs](https://playwright.dev/docs/test-advanced#launching-a-development-web-server-during-the-tests). I simplified config a bit.